src/main/resources/swagger/api-docs.yaml

```diff
@@ -4115,10 +4115,30 @@ definitions:
       ok:
         type: boolean
         description: whether any system(s) need attention
-
+        default: false
```

Comment:
The idea is that because the `systems` structure includes a failing system, the `ok` boolean should be set to `false`.

I agree that `example` is correct versus `default`, we seem to make that mistake a lot actually.